src/components/ImageView/index.tsx

```diff
@@ -6,31 +7,51 @@ import styles from './styles.module.scss';
 type PublicProps = {
   _btoa?: typeof btoa;
   _log?: typeof log;
+  _sanitize?: typeof purify.sanitize;
   mimeType: string;
   content: string;
 };
 
 type ConvertImageDataParams = {
   _btoa?: typeof btoa;
   _log?: typeof log;
+  _sanitize?: typeof purify.sanitize;
   content: string;
+  mimeType: string;
 };
 
-export const convertImageData = ({
+const convertImageData = ({
   _btoa = btoa,
   _log = log,
+  _sanitize = purify.sanitize,
   content,
+  mimeType,
 }: ConvertImageDataParams): string | null => {
   try {
-    return _btoa(content);
+    return _btoa(
+      // Run sanitize on svg files.
+      mimeType === 'image/svg+xml' ? _sanitize(content) : content,
```

Comment:
The security of this sanitization relies on `mimeType`. I think that's probably OK but if someone could figure out a way to set the mime type to `text/html` then that would result in an XSS'able data URL (maybe). I know it sounds paranoid but I think it wouldn't hurt to just add a simple check right above this line, like:

```tsx
if (!mimeType.startsWith('image')) {
  throw new Error(`Unexpected mimeType: "${mimeType}"`);
}
```